packages/endpoints-organizations-registrar/src/entities/signatories/factories/signatory-status-factory.js

```diff
@@ -33,9 +34,10 @@ module.exports = (app) =>
 
       return {
         organizationDid: organization.didDoc.id,
+        organizationId: new ObjectId(organization._id),
         events: [
           {
-            state: SignatoryEventStatus.EMAIL_SENT,
+            state: SignatoryEventStatus.LINK_SENT,
```

Comment:
why do we still need EMAIL_SENT?

Reply:
In order to track when last email was sent, to know whether or not need to send more. Are you thinking that with expiration this wouldn't be needed?